bin/ch/ch.cpp

```diff
@@ -521,7 +521,7 @@ int _cdecl wmain(int argc, __in_ecount(argc) LPWSTR argv[])
     HINSTANCE chakraLibrary = nullptr;
     bool success = ChakraRTInterface::LoadChakraDll(&argInfo, &chakraLibrary);
 
-#if defined(CHAKRA_STATIC_LIBRARY)
+#if defined(CHAKRA_STATIC_LIBRARY) && !defined(NDEBUG)
```

Comment:
Prefer `&& ENABLE_DEBUG_CONFIG_OPTIONS`. That's used everywhere to represent DEBUG and TEST build types.